### ISSUE TITLE
creates code in chat answer instead of files lol

### ISSUE THREAD

**Issue by ADS-1337** [stackblitz/bolt.new#3893]
### Describe the bug

started writing code in chat instead of updating files. No files updated at the end
<img width="426" alt="Screenshot 2024-12-08 at 9 40 56 PM" src="https://github.com/user-attachments/assets/95699aac-f32a-426c-9320-b9ab55785a68">

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-ssefdxxu

### Steps to reproduce

use ~130m tokens in 1 chat

### Expected behavior

to update files, not provide me with code

### Platform

```
Browser name = Chrome
Full version = 131.0.0.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 161237497,
  "usedJSHeapSize": 130533657,
  "jsHeapSizeLimit": 2197815296
}
Username = ADS-1337
Chat ID = ce99e7c90414
```

**Comment by mbhvl** [comment]
Yes seen this many times.  Instead of making code-changes it just talks about making code-changes.  Refreshing the browser containing BOLT.new usually fixes this but it is an enormous waste of tokens.
